src/views/admin/oneorder.ejs

```diff
@@ -9,132 +9,80 @@
               <div class="card-header py-3">
                 <div class="list-group-item d-flex justify-content-between align-items-center border-0 px-0 pb-0">
                   <h5>Order</h5>
-                  <span class="badge bg-warning text-dark">Pending</span>
+                  <% let status=order.status %>
+                    <%if (status=='pending' ) { %>
+                      <span class="badge bg-warning text-dark">Pending</span>
+                      <% } else if (status=='done' ) { %>
+                        <span class="badge bg-success">Done</span>
+                        <% } %>
                 </div>
               </div>
               <div class="card-body">
+                <% products.forEach(product=> { %>
+                  <div class="row">
+                    <div class="col-lg-3 col-md-12 mb-4 mb-lg-0">
+
+                      <div class="bg-image hover-overlay hover-zoom ripple rounded" data-mdb-ripple-color="light">
+                        <% let photoPath='/laptom_img/' + product.Photo_Path + ".webp" %>
```

Comment:
zdjęcia nie wyświetlają się.  Pamiętajmy żeby to później naprawić.

Reply:
Chwilowo tak jest, bo w repo_demo nie mamy podanego Photo_path, dodajac produkty do faktycznej bazy danych trzeba to dodac